pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[build-system]
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "breathing_cat"
+version = "0.1.0"
+description = "Tool to build documentation for C++/Python-Packages used at MPI-IS"
+authors = [
+    {name = "Maximilien Naveau"},
+    {name = "Felix Widmaier"},
+]
+maintainers = [
+    {name = "Felix Widmaier", email = "felix.widmaier@tuebingen.mpg.de"}
+]
+readme = "README.md"
+license = {file = "LICENSE"}
+
+dependencies = [
```

Comment:
doxygen?

Reply:
Doxygen is not a Python package (i.e. cannot be installed by pip), so I think it shouldn't be listed here.
Unfortunately, I don't see a better way than asking the user to install it manually.

Reply:
It currently will show an error at runtime if doxygen is not found, including an `apt` command to install it.  I think this should be enough.